test/core/network/httpRouter/httpRouter.test.js

```diff
@@ -623,5 +624,20 @@ describe('core/network/httpRouter', () => {
         }
       });
     });
+
+    it('deprecated method should have good properties', done => {
```

Comment:
You don't need to use the `done` method I think. The test will pass if the function does not throw